app/helpers/template_paths_helper.rb

```diff
@@ -0,0 +1,14 @@
+module TemplatePathsHelper
+  def template_path_for(type, route = 'path')
```

Comment:
Someone might want to use `url` instead of `path` later, I expect more templates type to extend this method so I wanted to provide this option, on the other hand `template_path_for` maybe misleading name, since it can return path or url. Suggestions?